content/support-training.html

```diff
@@ -209,6 +209,13 @@ <h2>Commercial support</h2>
               <img src="../assets/commercial-support-logos/xamira_networks.png" alt="xamira networks">
             </a>
           </div>
+
+        <div class="col-md-3 col-sm-6 doc-content tc-col">
+            <a href="https://www.asyncdatado.com/" class="btn btn-training btn-default btn-large" target="_blank">
+              <img src="../assets/commercial-support-logos/asyncdatado.png" alt="asyncdata do dominicana">
```

Comment:
I see you've created a second PR (https://github.com/prometheus/docs/pull/2423) for the logo file, but we will want it in the same PR so we can see the deploy preview site with the logo. Could you please add the logo to this PR?